cla_backend/apps/cla_butler/management/commands/monitor_missing_outcome_codes.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+import logging
+from django.core.management.base import BaseCommand
+
+from cla_butler.stack import is_first_instance, InstanceNotInAsgException, StackException
+
+from cla_eventlog.constants import LOG_LEVELS, LOG_TYPES
+from cla_eventlog.models import Log
+from legalaid.models import Case
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Count and alert when outcome codes denormalized to Case instances are missing'
```

Comment:
🙋‍♂️ This description and the surrounding code can be a bit hard to understand if someone is not involved in the missing outcome codes ticket.

Someone might look at logs and see a warning that says `Cases found missing denormalized outcome codes` and might be wondering if that's a problem or not (might be a red herring with live issues).

What do you think about rephrasing to something like "Found high importance cases with mismatching outcome codes; the outcome should have been saved on the case too"?

Reply:
Reworded to reference the ticket, the code's temporary investigative nature, and for clarity.